modules/EnsEMBL/Web/Object.pm

```diff
@@ -355,7 +355,7 @@ sub get_slices {
       slice             => $_,
       underlying_slices => $underlying_slices && $_->can('get_all_underlying_Slices') ? $_->get_all_underlying_Slices : [ $_ ],
       name              => $name,
-      display_name      => $self->get_slice_display_name($self->hub->species_defs->production_name_mapping($name), $_),
+      display_name      => $self->get_slice_display_name($name, $_),
```

Comment:
are we sure `$name` is the species url name here?

Reply:
Yes, this is another place where I had to make changes because the web display was wrong.